tests/modules/custom-status/meta/test-required-metadata-id-handler.php

```diff
@@ -22,12 +23,20 @@ protected function setUp(): void {
 		// Normally custom statuses are installed on 'admin_init', which is only run when a page is accessed
 		// in the admin web interface. Manually install them here. This avoid issues when a test creates or deletes
 		// a status and it's the only status existing, which can cause errors due to status restrictions.
-		VIP_Workflow::instance()->custom_status->install();
+		CustomStatus::setup_install();
+	}
+
+	// tear down after class
+	protected function tearDown(): void {
+		parent::tearDown();
+
+		// Reset all module options
+		OptionsUtilities::reset_all_module_options();
```

Comment:
Due to our new static refactoring, this is necessary to do. It ensures each test doesn't conflict with the other. Otherwise, one test could trigger a module's install step and the other one wouldn't be able to do the same as it's already done.